src/pybind/mgr/telemetry/module.py

```diff
@@ -204,6 +211,13 @@ def gather_crashinfo(self):
                 continue
             c = json.loads(crashinfo)
             del c['utsname_hostname']
+            (etype, eid) = c.get('entity_name', '').split('.')
+            if etype != 'osd':
+                m = hashlib.sha1()
+                m.update(self.salt.encode('utf-8'))
+                m.update(eid.encode('utf-8'))
+                m.update(self.salt.encode('utf-8'))
```

Comment:
twice for fun, or typo?

Reply:
i seem to recall this is a standard pattern when doing HMAC to put the salt or hash both before and after as a confounder.  probably doesn't really matter.

Reply:
mostly for fun i guess :)